liquidcore/config/serializers.py

```diff
@@ -0,0 +1,124 @@
+import re
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+from .models import *
+
+def valid_username(username):
+    if not username:
+        raise serializers.ValidationError("empty username")
+    if not bool(re.match(r"^(\w|\.)+$", username)):
+        raise serializers.ValidationError("username can only contain letters, digits, underscore and dot")
+    if len(username) > 64:
+        raise serializers.ValidationError("username too long")
+    return username
+
+def valid_password(password):
+    if len(password) < 8:
+        raise serializers.ValidationError("password too short")
+    return password
+
+def valid_domain(domain):
+    if not bool(re.match(r"^((\w|-)+\.)+(\w+)$", domain)):
```

Comment:
I think `_` is a valid character too. There are probably others, that we don't care about, but `_` might actually be used.

Reply:
`\w` is alphanumeric + underscore. The "alpha" part permits unicode word characters, so I think we want to limit it to the ASCII `[a-zA-Z0-9_]` equivalent for now.  See https://docs.python.org/3.6/library/re.html for details

Reply:
Yes.